configure.ac

```diff
@@ -2,6 +2,7 @@ dnl Process this file with autoconf to produce a configure script.
 
 AC_PREREQ(2.57)
 AC_INIT([libfabric], [0.0.2], [ofiwg@lists.openfabrics.org])
+AC_SUBST(RPM_VERSION, AC_PACKAGE_VERSION)
```

Comment:
They would be the same except that the RPM version cannot contain dashes.

If we wanted them to be consistent, we could just transliterate dashes into dots in the tarball version as well.